### PR TITLE
Add stat eng script and improve README rendering

### PR DESCRIPTION
**Pull request by gravesti**
* Adds code to reproduce Statistical Engineering hex
* Refresh svg and png based on that script
* Remove fixed "width" attribute from thumbnails in table since not all files are square

### CODE REVIEW

File: OTHER_FORMATS/StatisticalEngineering.R [hunk 1]
```diff
@@ -0,0 +1,104 @@
+# Creates a Statistical Engineering Hex Sticker
```
Comment (danielinteractive) [inline OTHER_FORMATS/StatisticalEngineering.R:1]: maybe rename folder to `R`? or `scripts`?

File: OTHER_FORMATS/StatisticalEngineering.R [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      s1,
+      parabola(s2),
+      parabola(s3),
+      s4
+    ),
+    w = c(seq(4, 1, length = 4), rep(1, 32), seq(1, 4, length = 4))
+  )
+
+feature_color <- "#1881C2"
+bridge_fill <- "#BFDBED"
```
Comment (danielinteractive) [inline OTHER_FORMATS/StatisticalEngineering.R:68]: somehow the bridge fill was a tiny bit darker before? I mean does not matter that much but it is nice to see the CI a bit clearer?

**Review by danielinteractive — COMMENTED** [review]
thanks @gravesti, nice, 2 minor comments inside

File: OTHER_FORMATS/StatisticalEngineering.R (hunk 2 above)
Comment (gravesti) [reply OTHER_FORMATS/StatisticalEngineering.R:68]: Fixed. A mix up with ggplot, transparency and the macos color meter :)

**Review by danielinteractive — APPROVED** [review]

**Pull request merged**